src/client/components/MainWindow/index.jsx

```diff
@@ -1,3 +1,8 @@
+import { connect } from 'react-redux';
 import MainWindow from './MainWindow';
 
-export default MainWindow;
+const mapStateToProps = state => ({
```

Comment:
You can also (not required) use short record with destructuring:
```js
const mapStateToProps = ({ showPage }) => ({ showPage });
```

Reply:
I don't understand how we use it here. but it works. 
done according to your comment.